style.scss

```diff
@@ -0,0 +1,388 @@
+}
+
+.introduction-me__job--text {
+    font-size: $mainFontSize;
+    font-style: $nextInformation;
+    margin: 0;
+}
+.introduction__portrait--modification {
+    width: 100px;
+    border-radius: 50%;
+    justify-self: center;
+    order: -1;
+    margin-top: 1.5rem;
+    margin-bottom: 1rem;
+}
+
+
+.contact-icons { 
+    padding-left: 0;
+    display: flex;
+    list-style: none;
+    margin-top: $bigMargin;
+    justify-content: space-evenly;
+}
+
+.contact-icons__button {
+    padding: 0.3rem;
+}
+
+.contact-icons__button--modification {
+    height: 20px;
+}
+
+/*---HEADING---*/
+
+.section_heading {
+    padding: $mainPadding 2rem;
+    background-image: $mainGradient;
+    text-align: center;
+
+    &--text {
+        font-size: $mainFontSize;
+        font-weight: $importantFont;
+    }
+
+}
+
+
+/*---ABOUT ME---*/
+
+.about-me {
+    margin: $bigMargin;
+
+    &__description {
+        margin-bottom: $bigMargin;
+        text-align: center;
+
+        &--text {
+            font-size: $smallFontSize;
+        }
+    }
+}
+
+.about-me-timeline-point {
+    margin-bottom: $bigMargin;
+}
+
+.about-me-timeline-point-description {
+    @include flexColumn ();
+    align-items: center;
+    text-align: center;
+}
+
+.about-me-timeline-point-description__year--text {
+    font-size: 0.9rem;
+    font-weight: $importantFont;
+}
+
+.about-me-timeline-point-description__info--text {
+    font-size: $smallFontSize;
+    font-style: $nextInformation;
+}
+
+.about-me-timeline-point img {
+    display: none;
+}
+
+/*---PORTFOLIO---*/
+
+
+.portfolio {
+    display: grid;
+    justify-items: center;
+    row-gap: 1.5rem;
+    margin: $bigMargin;
+    
+    &__description {
+        text-align: center;
+    }
+}
+
+
+.nabidka {
+        img {
+            place-self: center;
+            width: 100%;
+            height: 100%;
+}
+}
+
+
+/*---CONTACT---*/
+
+.contact {
+    padding: $mainPadding;
+    text-align: center;
+}
+
+.contact--text {
+    font-size: $smallFontSize;
+}
+
+.contact__address--text {
+    line-height: 1.2rem;
+    font-style: $nextInformation;  
+}
+
+
+/*---tablet---*/
+
+@media screen and (min-width: 450px) {
+
+
+/*---HEADER---*/
+
+header {
+    padding: $mainPadding;    
+}
+
+.header__logo--modification {
+    height: 35px;
+}
+
+.nav--text {
+    font-size: $mainFontSize;
+}
+
+/*---MAIN INFORMATION---*/
+
+.introduction {
+    margin: $bigMargin;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.introduction-me {
+    padding-right: 0.5rem;
+    flex-basis: 60%;
+    order: -1;
+}
+
+/*Zde bych se chtěla zeptat, zda můžu udělat pouze jednokrokové zanoření, tedy &__year--text nebo to musím rozeptat a --text jakožto modifikátor elementu year dát jako další zanoření? Jde mi o to, že vlastnost mám až pro modifikátor elementu ne pro element samotný a pokud bych to musela zanořovat ve dvou krocích bylo by to více řádků než v původní podobě. Snad jsem to napsala als trochu pochopitelně :) děkuji*/
```

Comment:
Můžeš udělat i "jednokrokové zanoření", to není problém. :3 Víc vše rozepisovat/odřádkovávat/zanořovat je záležitost čitelnosti, kódu je to jedno. Ve spoustě jazyků bys mohla veškerý kód napsat na jeden řádek a stejně by fungoval, různě odřádkováváme a zanořujeme primárně pro čitelnost z našeho pohledu a často si to tak můžeme přizpůsobit konkrétnímu příkladu.